### ISSUE TITLE
Unexpected token '>' when running uglifyJS

### ISSUE THREAD

**Issue by caisah** [node-xmpp/ltx#105]
Hi, 

I'm using a library which depends on **lxt** (https://github.com/otalk/jxt) and when I run _UglifyJS_ through _webpack_ on my build I get:
```shell
ERROR in 3-chunk41c1d9e763e90e289c4a.js from UglifyJs

SyntaxError: Unexpected token: operator (>) [./~/ltx/lib/stringify.js:9,0][3-chunk41c1d9e763e90e289c4a.js:44960,37]
```

This worked well until yesterday and I was able to build just fine. Do you think it has something to do with the latest changes?

Thanks!

**Comment by sonnyp** [comment]
probably introduced with https://github.com/node-xmpp/ltx/pull/104/files but looks like an issue with uglify

could you try with latest uglify version ?

**Comment by billybooth** [comment]
I'm getting the same today with node-xmpp's examples.

**Comment by fecaleagle** [comment]
I'm getting the same today with the node-xmpp examples.

**Comment by sonnyp** [comment]
https://github.com/node-xmpp/ltx/pull/106

**Comment by sonnyp** [comment]
released as 2.6.1

**Comment by caisah** [comment]
Thanks for the quick update!
